azurerm/internal/services/network/point_to_site_vpn_gateway_resource.go

```diff
@@ -103,6 +145,46 @@ func resourceArmPointToSiteVPNGateway() *schema.Resource {
 				ValidateFunc: validation.IntAtLeast(0),
 			},
 
+			"custom_dns_servers": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.IsIPv4Address,
+				},
+			},
+
+			"connection_health": {
```

Comment:
i don't think any of this makes sense in terraform so could we remove them? they will constantly be changed